source/includes/steps-starting-compass-individual-fields.yaml

```diff
@@ -222,17 +222,16 @@ content: |
 
                  Select :guilabel:`X.509` if the deployment uses
                  :manual:`X.509 </core/security-x.509/>` as its
-                 authentication mechanism. If selected, you must
-                 provide the  :guilabel:`Username` to authenticate
-                 the user.
+                 authentication mechanism. 
 
+                 If selected, you may optionally provide the 
+                 :guilabel:`Username` to authenticate the user.
+                 
                  .. note::
 
-                    If you are using 
-                    :atlas:`Atlas-managed certificates </security-add-mongodb-users/#select-an-authentication-method>`
-                    , your username must be prefaced by "CN="
-                    per :rfc:`2253`. For example, the username 
-                    "X509User" must be provided as "CN=X509User".
```

Comment:
This part about Atlas certificates I think we should keep. When a user does provide a username with Atlas certificates it needs the subject `CN=` prefix.

Reply:
Gotcha, adding that back in